docker/Dockerfile

```
@@ -1,19 +1,14 @@
+# To build this container, go to ESMValCore root folder and execute :
+# docker build -t ${TAG_OF_YOUR_CHOICE} . -f docker/Dockerfile
 FROM continuumio/miniconda3
 
-# update the conda packages
-RUN conda update -y conda pip
+COPY . /src/ESMValTool
+WORKDIR /src/ESMValTool
 
-# install development tools
-RUN apt-get update -y && apt-get install -y \
-    build-essential \
-    curl \
-    unzip
-
-# install environment packages
-RUN conda install -c conda-forge -c esmvalgroup -c birdhouse esmvaltool
-
-# run tests
-RUN esmvaltool -h
+RUN conda update -y conda pip && conda env update --name base --file environment.yml && conda clean --all -y
+RUN pip install . && pip cache purge
+RUN apt install julia -y && julia esmvaltool/install/Julia/setup.jl
+RUN Rscript esmvaltool/install/R/setup.R
 
 ENTRYPOINT ["esmvaltool"]
-CMD ["-h"]
+CMD ["--help"]
```

```suggestion
CMD ["-- --help"]
```
would be nicer, because it avoids the message
```
INFO: Showing help with the command 'esmvaltool -- --help'.
```
at the top of the output